stdlib/public/core/UnicodeScalarProperties.swift

```diff
@@ -52,20 +52,14 @@ extension Unicode.Scalar {
 /// Boolean properties that are defined by the Unicode Standard (i.e., not
 /// ICU-specific).
 extension Unicode.Scalar.Properties {
-  internal func _hasBinaryProperty(
-    _ property: __swift_stdlib_UProperty
-  ) -> Bool {
-    return __swift_stdlib_u_hasBinaryProperty(icuValue, property) != 0
-  }
-
   /// A Boolean value indicating whether the scalar is alphabetic.
   ///
   /// Alphabetic scalars are the primary units of alphabets and/or syllabaries.
   ///
   /// This property corresponds to the "Alphabetic" property in the
   /// [Unicode Standard](http://www.unicode.org/versions/latest/).
   public var isAlphabetic: Bool {
-    return _hasBinaryProperty(__swift_stdlib_UCHAR_ALPHABETIC)
+    _swift_stdlib_hasBinaryProperty(_scalar.value, 1 << 7)
```

Comment:
Hey now, you know you're not going to be able to merge with a bunch of magic numbers and shifts here :-)